conf/messages.en

```diff
@@ -1735,4 +1735,13 @@ raRAllowancesCYA.heading = raRAllowancesCYA
 haveYouFinishedThisSection.heading = Have you finished this section?
 haveYouFinishedThisSection.title = Have you finished this section?
 haveYouFinishedThisSection.hint = You’ll still be able to go back and review the information that you ve given us once you’ve ve finished.
-haveYouFinishedThisSection.error.required = Select yes if you have completed this section
\ No newline at end of file
+haveYouFinishedThisSection.error.required = Select yes if you have completed this section
+
+ukrentaroom.raRAdjustmentsComplete.title = Have you completed this section?
```

Comment:
Lets not duplicate these common texts  (Rebase with main)

Reply:
Rebased and fixed